less/home-page.less

```diff
@@ -17,19 +26,32 @@
   .space-images {
     display: flex;
     flex-wrap: wrap;
-    width: 100%;
+    .full();
+
+    @media @phone {
+      & :not(:first-child) {
```

Comment:
This is a really awesome way of getting the other images to disappear. Really cool!